src/components/Reports/GoalCalculator/SharedComponents/GoalCalculatorGrid/GoalCalculatorGrid.test.tsx

```diff
@@ -118,12 +118,17 @@ describe('GoalCalculatorGrid', () => {
     });
 
     await waitFor(() => {
-      expect(mutationSpy).toHaveGraphqlOperation('DeleteSubBudgetCategory', {
-        input: {
-          accountListId: 'account-list-1',
-        },
-      });
+      expect(mutationSpy).toHaveGraphqlOperation('DeleteSubBudgetCategory');
```

Comment:
If you change the component to

```ts
const tempId = `temp-${DateTime.now().toISO()}`;
```

then we will know the timestamp since that is mocked in `__tests__/util/setup.ts`. And you could write this as simply:

```ts
    await waitFor(() => {
      expect(mutationSpy).toHaveGraphqlOperation('DeleteSubBudgetCategory', {
        input: {
          id: 'temp-2020-01-01T00:00:00.000+00:00',
          accountListId: 'account-list-1',
        },
      });
    });
```